src/upstream_patches_ui.sh

```diff
@@ -193,6 +200,35 @@ function show_bookmarked_series_details()
   esac
 }
 
+# Screen that shows all types of settings available.
+function show_settings_screen()
+{
+  local -a menu_list_string_array
+  local message_box
+  local ret
+
+  menu_list_string_array=('Mailing Lists')
+  message_box=''
+  create_menu_options 'Settings' "$message_box" 'menu_list_string_array' 1
+  ret="$?"
+
+  case "$ret" in
+    0) # OK
+      case "$menu_return_string" in
+        1) # Mailing Lists
+          screen_sequence['SHOW_SCREEN']='register'
```

Comment:
If I follow: Setting > Mailing Lists, I'll only have `OK` and `EXIT` options which make sense for the first screen. If I choose the OK option, it will send me back to the dashboard, which is not intuitive. It is time to parameterize the `register` option to enable it to return to the previous menu.